releasenotes/notes/0.19/retworkx-substitute_node_with_dag-speedup-d7d1f0d33716131d.yaml

```diff
@@ -4,6 +4,6 @@ features:
     Various transpilation internals now use new features in `retworkx
     <https://github.com/Qiskit/retworkx>`__ 0.10 when operating on the internal
     circuit representation.  This can often result in speedups in calls to
-    :obj:`~qiskit.transpile` of around 10-40%, with greater effects at higher
+    :obj:`~.compiler.transpile` of around 10-40%, with greater effects at higher
     optimisation levels.  See `#6302
```

Comment:
```suggestion
    optimization levels.  See `#6302
```

Reply:
It shows that I'm the one who edited these, huh?

Reply:
Done in e624bd6.